source/js/buyers-guide/search/creep-utils.js

```diff
@@ -18,8 +18,24 @@ export class CreepUtils {
     const container = document.querySelector(`.product-box-list`);
     const list = [...container.querySelectorAll(`.product-box`)];
     const creepVal = (e) => parseFloat(e.dataset.creepiness);
-    list
-      .sort((a, b) => creepVal(a) - creepVal(b))
-      .forEach((p) => container.append(p));
+    const alphaSort = (e) => e.querySelector(".product-name").innerText;
```

Comment:
What do you think about using function names that follow the [A/HC/LC pattern](https://github.com/kettanaito/naming-cheatsheet#ahclc-pattern) (starting with an action followed by a object that is being acted on).

Reply:
Sure never seen this document but I can try to rename it with this in mind